src/internal/Subscriber.ts

```diff
@@ -83,7 +83,7 @@ export class Subscriber<T> extends Subscription implements Observer<T> {
    * @return {void}
    */
   error(err?: any): void {
-    if (this.isStopped) {
+    if (this.isStopped && !config.useDeprecatedSynchronousErrorHandling) {
```

Comment:
@leggechr this is the fix.

Reply:
Surely this is going to cause other problems? This renders stopped meaningless for error notifications if that config is used? How did this ever work in v6? If it didn't, should we really change this? FWIW, I don't particularly care, but I thought I'd point it out.